apps/extension/src/core/domains/sitesAuthorised/store.ts

```diff
@@ -1,6 +1,6 @@
 import { AuthorizedSite, AuthorizedSites, ProviderType } from "@core/domains/sitesAuthorised/types"
 import { SubscribableByIdStorageProvider } from "@core/libs/Store"
-import { isTalismanHostname } from "@core/page"
+import { isTalismanHostname } from "@core/util/isTalismanHostname"
```

Comment:
These utilities were extracted from `page.ts` because importing from `page.ts` caused all the code in the file to be executed in the test environment.